freqtrade/persistence.py

```diff
@@ -25,15 +25,14 @@
 _SQL_DOCS_URL = 'http://docs.sqlalchemy.org/en/latest/core/engines.html#database-urls'
 
 
-def init(config: Dict) -> None:
+def init(db_url: str, dry_run: bool = False) -> None:
```

Comment:
same here. if there should be a default value, then I don't see why it should be False

Reply:
This was intentional - and looking at the meaning of dry-run for this function it does also make sense (if it's True, it'll close all open orders).